src/Calendar.js

```diff
@@ -123,6 +123,15 @@ class Calendar extends React.Component {
     onSelectEvent: PropTypes.func,
 
     /**
+     * Callback fired when a calendar event is clicked twice.
+     *
+     * ```js
+     * (event: Object, e: SyntheticEvent) => any
```

Comment:
should be `=> void` for the return type i believe

Reply:
@jquense done